tests/xtimer_msg_receive_timeout/main.c

```diff
@@ -30,27 +26,27 @@
 #include "xtimer.h"
 #include "timex.h"
 
-#define TEST_PERIOD (100000LU)
+#define TEST_PERIOD     (1*US_PER_SEC) /* 1 sec in US */
```

Comment:
hey, only have an _duemilanove_ at hand. However, that works with 100ms, too. Drift in output timing is around 10ms late from the expected timestamp, but works.